### PR TITLE
minor corrections in amss2.rst

### PR DESCRIPTION
**Pull request by AnjuJoon**

### CODE REVIEW

File: source/rst/amss2.rst [hunk 1]
```diff
@@ -12,7 +12,7 @@ Fluctuating Interest Rates Deliver Fiscal Insurance
 
 **Co-authors: Anmol Bhandari and David Evans**
 
-In addition what's in Anaconda, this lecture will need the following libraries
+In addition, what's in Anaconda, this lecture will need the following libraries
```
Comment (jstac) [inline source/rst/amss2.rst]: Please change to "In addition to what's in..."

(Are there other lectures with this issue?)

File: source/rst/amss2.rst [hunk 2]
```diff
@@ -109,12 +109,12 @@ For a long time, the Ramsey plan puts a nontrivial martingale-like component int
 smooths distortions from the labor tax rate across  time and Markov states
 
 But BEGS show that binding implementability constraints slowly push government debt in a direction designed to let the government use fluctuations in equilibrium interest
-rate  rather than fluctuations in  par values of debt to insure against shocks to government expenditures
+rate  rather than fluctuations in  par values of debt to ensure against shocks to government expenditures
```
Comment (jstac) [inline source/rst/amss2.rst]: undo

File: source/rst/amss2.rst [hunk 3]
```diff
@@ -76,9 +76,9 @@ This lecture studies a special  AMSS model in which
 
    - the **market value** :math:`\frac{\bar b}{R_t(s_t)}` of government debt at time :math:`t`  varies as a time-invariant function of the Markov state :math:`s_t`
 
-   - fluctuations in the interest rate make gross earnings on government debt :math:`\frac{\bar b}{R_t(s_t)}` fully insure the gross-of-gross-interest-payments government budget against fluctuations in government expenditures
+   - fluctuations in the interest rate make gross earnings on government debt :math:`\frac{\bar b}{R_t(s_t)}` fully ensure the gross-of-gross-interest-payments government budget against fluctuations in government expenditures
```
Comment (jstac) [inline source/rst/amss2.rst]: undo

File: source/rst/amss2.rst [hunk 4]
```diff
@@ -109,12 +109,12 @@ For a long time, the Ramsey plan puts a nontrivial martingale-like component int
...
 
 But BEGS show that binding implementability constraints slowly push government debt in a direction designed to let the government use fluctuations in equilibrium interest
-rate  rather than fluctuations in  par values of debt to insure against shocks to government expenditures
+rate  rather than fluctuations in  par values of debt to ensure against shocks to government expenditures
 
 - This is a **weak** (but unrelenting) force that, starting from an initial debt level, for a long time is dominated by the stochastic martingale-like component of debt
   dynamics that the Ramsey planner uses to facilitate imperfect tax-smoothing across time and states
 
-- This weak force slowly drives the par value of government **assets** to a **constant** level at which the government can completely insure against government expenditure shocks while
+- This weak force slowly drives the par value of government **assets** to a **constant** level at which the government can completely ensure against government expenditure shocks while
```
Comment (jstac) [inline source/rst/amss2.rst]: undo

File: source/rst/amss2.rst [hunk 5]
```diff
@@ -212,7 +212,7 @@ Specification with CRRA Utility
 
 
 As in lectures :doc:`optimal taxation without state-contingent debt<amss>` and :doc:`optimal taxation with state-contingent debt<opt_tax_recur>`,
-we assume that the  representative agent has  utility function
+we assume that the representative agent has utility function:
```
Comment (jstac) [inline source/rst/amss2.rst]: please undo

File: source/rst/amss2.rst [hunk 6]
```diff
@@ -255,7 +255,7 @@ and
 
     (1+\Phi) [u_c(c_0) + u_n(c_0+g_0)] + \Phi[c_0 u_{cc}(c_0) + (c_0+g_0) u_{nn}(c_0+g_0)] - \Phi u_{cc}(c_0) b_0 = 0
 
-In equation :eq:`amss2_TS_barg10`, it is understood that :math:`c` and :math:`g` are each functions of the Markov state :math:`s`
+In equation :eq:`amss2_TS_barg10`, it is understood that :math:`c` and :math:`g` are each function of the Markov state :math:`s`
```
Comment (jstac) [inline source/rst/amss2.rst]: please undo

File: source/rst/amss2.rst [hunk 7]
```diff
@@ -654,7 +654,7 @@ BEGS also derive the following  approximation to the rate of convergence to :mat
 Formulas and Code Details
 --------------------------
 
-For our example, we describe some code that we use to compute the steady state mean and the rate of convergence to it
+For our example, we describe some code that we use to compute the steady-state mean and the rate of convergence to it
```
Comment (jstac) [inline source/rst/amss2.rst]: please undo

File: source/rst/amss2.rst (hunk 1 above)
Comment (AnjuJoon) [reply source/rst/amss2.rst]: yes, there are some lectures in which it is - In addition to what's in.. 
and in some, it is as in this lecture.

**Pull request merged**